examples/layer-browser/webpack.config.js

```diff
@@ -20,7 +20,8 @@ const CONFIG = {
     alias: {
       // From mapbox-gl-js README. Required for non-browserify bundlers (e.g. webpack):
       'mapbox-gl$': resolve('./node_modules/mapbox-gl/dist/mapbox-gl.js'),
-      '@math.gl/web-mercator': resolve('../../node_modules/@math.gl/web-mercator')
+      '@math.gl/web-mercator': resolve('../../node_modules/@math.gl/web-mercator'),
+      '@mapbox/tiny-sdf': resolve('../../node_modules/@mapbox/tiny-sdf')
```

Comment:
I think you can drop this now that the dependency conflict is fixed

Reply:
Without this line the `layer-browser` example doesn't seem to work for me as it ends up using `@mapbox/tiny-sdf` version `1.2.5` instead of `2.0.5`.

`yarn why @mapbox/tiny-sdf` shows it's because of a conflict with `react-map-gl`.

```
=> Found "@mapbox/tiny-sdf@1.2.5"
info Reasons this module exists
   - "react-map-gl#mapbox-gl" depends on it
   - Hoisted from "react-map-gl#mapbox-gl#@mapbox#tiny-sdf"
```